Project/Source/Panels/PanelGameControllerDebugging.cpp

```diff
@@ -0,0 +1,128 @@
+#include "PanelGameControllerDebugging.h"
+
+#include "Application.h"
+#include "Modules/ModuleEditor.h"
+#include "Modules/ModuleInput.h"
+
+#include "IconsForkAwesome.h"
+
+#include "Utils/Leaks.h"
+
+#define N_PLAYERS 2
+
+PanelGameControllerDebugging::PanelGameControllerDebugging()
+	: Panel("Game Controller Debugger", false) {
+}
+
+void PanelGameControllerDebugging::Update() {
+	ImGui::SetNextWindowSize(ImVec2(400.0f, 200.0f), ImGuiCond_FirstUseEver);
+	std::string windowName = std::string(ICON_FK_INFO_CIRCLE " ") + name;
+	if (ImGui::Begin(windowName.c_str(), &enabled, ImGuiWindowFlags_AlwaysAutoResize)) {
+		for (int i = 0; i < N_PLAYERS; ++i) {
+			DisplayGamepadInfo(i, App->input->GetPlayerController(i));
+			ImGui::Separator();
+		}
+	}
+	ImGui::End();
+}
+
+void PanelGameControllerDebugging::DisplayGamepadInfo(int index, PlayerController* playerController) {
+	ImGui::TextColored(App->editor->textColor, "Game Controller %d:", index);
+	ImGui::SameLine();
```

Comment:
This may be better suited to the configuration panel. ¿Is there a reason this has to be in a separate panel?

Reply:
Renamed to PanelDebug